### PR TITLE
Moved everything into main README for Themer Documentation

### PR DESCRIPTION
**Pull request by MrSpidercat**
Moved everything into main README file, fixed tons of grammar errors, and more.

### CODE REVIEW

**Comment by MrSpidercat** [comment]
Okay, I'm done transferring information over and fixing some things. 
MERGE MERGE MERGE MERGE MERGE MERGE MERGE

**Comment by Reinified** [comment]
merge request 9 now now now now now

File: theme-dev/README.md [hunk 1]
```diff
@@ -1,33 +1,295 @@
...
+
+* As Riyu has found out, a lot of things in Light mode are not themable, but ven did say he will add more light mode support. [Link to Riyu's light mode tests](https://ptb.discord.com/channels/811255666990907402/868419532992172073/902278346195476490)
+
+
+
+
+<br />
+<br />
 
-<h1 align="Center">Credits</h1>
```
Comment (ItzOnlyAnimal) [inline theme-dev/README.md:27]: Add the credits back

File: theme-dev/README.md [hunk 2]
```diff
@@ -1,33 +1,295 @@
...
+### Creating a Simple Theme
  
-  * Start by making a new theme inside of the themer plugin settings, give it a name and it will set version and author automatically for you.
+  * Start by making a new Theme inside of the Themer Plugin settings, give it a name and it will set Version, and Author (You) automatically for you.
  * You will see multiple categories, choose the Simple Colors category and click on the + icon to add a new string. Add a `background` string and give it some color by clicking on it. 
  * Save, then select restart to see how it looks!
- * [Example Simple Colors Theme](https://cdn.discordapp.com/attachments/824357609778708580/865289689363251210/DiscordThemer_ZelkButBasic.json)
+
+[Download Example Theme](https://cdn.discordapp.com/attachments/824357609778708580/865289689363251210/DiscordThemer_ZelkButBasic.json)
 </details>
```
Comment (ItzOnlyAnimal) [inline theme-dev/README.md:40]: what is this here for :guh:

File: theme-dev/README.md [hunk 3]
```diff
@@ -1,33 +1,295 @@
...
+
+* github.com
+* raw.githubusercontent.com
+* gitlab.com
+* cdn.discordapp.com
+* media.discordapp.net
+* i.imgur.com
+* i.ibb.co
+
+<br />
```
Comment (ItzOnlyAnimal) [inline theme-dev/README.md:110]: i mightve missed it but why the random `<br>`s

**Review by ItzOnlyAnimal — CHANGES_REQUESTED** [review]
guh

File: theme-dev/README.md (hunk 2 above)
Comment (MrSpidercat) [reply theme-dev/README.md:40]: Show a way these strings can be used. I didn't add this specifc one though.

**Comment by GangsterFox** [comment]
Looks great! Will move when the pr is done
Kind of bummed to see that we moved back to one big readme but it's fine

**Comment by GangsterFox** [comment]
Should we add a troubleshooting part like for people who's chat bar doesnt get themed but can get fixed by enabling debuggable

Just an idea

**Comment by MrSpidercat** [comment]
> Looks great! Will move when the pr is done
> Kind of bummed to see that we moved back to one big readme but it's fine 
> 

I do think it's better though because it makes it easier for someone to just press Ctrl + F and find what they need to find

**Comment by MrSpidercat** [comment]
I think it's ready to be merged lmk if there's anything I may need to change.

**Comment by MrSpidercat** [comment]
merge when

**Review by MrSpidercat — COMMENTED** [review]
The mention in the name was accidental

**Comment by GangsterFox** [comment]
So this pr is ready for merge yes

**Review by mantikafasi — APPROVED** [review]

**Review by scrazzz — APPROVED** [review]

**Comment by Vendicated** [comment]
the table of contents is obsolete, this button exists
Screenshot_20211119-155833__01

**Comment by Reinified** [comment]
not everyone knows what that does so its not obsolete

**Comment by Vendicated** [comment]
"people dont know how to read title so add 3 of them"

**Comment by GangsterFox** [comment]
Still I can agree with soulz people are too stupid to know what that is and like shit spoon-fed

**Comment by TymanWasTaken** [comment]
Then replace table of contents with something like :press the button in the left corner for a table of contents"

**Comment by GangsterFox** [comment]
Tymon we have thousand guides but people still ask retarded question in support 
Do you think people will understand that one button instead of buttons straight in front of you explaining what part is what

**Comment by Reinified** [comment]
> "people dont know how to read title so add 3 of them"
what else would you expect from the support channels

**Comment by Vendicated** [comment]
this discussion is dumb lol. the table of contents is obsolete and hard to maintain (change a title, gotta change the table of contents). Add something saying click the button top right for table of contents and if someone doesnt understand that then thats their fault lol

**Comment by Reinified** [comment]
just keep the table of contents its useful for the kids who dont know how github works

**Comment by GangsterFox** [comment]
Especially how many 13 year olds we have making themes

**Comment by MrSpidercat** [comment]
I just feel it's better to have the table of contents because you're able to just see everything on your screen at once, no need to press another button to see the all the anchors. Most people don't know what that button is anyways

**Pull request merged**